src/abbfreeathome/const.py

```diff
@@ -21,11 +21,18 @@
 from .devices.room_temperature_controller import RoomTemperatureController
 from .devices.smoke_detector import SmokeDetector
 from .devices.switch_actuator import SwitchActuator
+from .devices.switch_actuator_virtual import SwitchActuatorVirtual
 from .devices.switch_sensor import DimmingSensor, SwitchSensor
 from .devices.temperature_sensor import TemperatureSensor
 from .devices.trigger import Trigger
 from .devices.wind_sensor import WindSensor
 from .devices.window_door_sensor import WindowDoorSensor
+from .devices.window_door_sensor_virtual import WindowDoorSensorVirtual
+
+FUNCTION_DEVICE_MAPPING_VIRTUAL: dict[Function, Base] = {
+    Function.FID_WINDOW_DOOR_SENSOR: WindowDoorSensorVirtual,
+    Function.FID_SWITCH_ACTUATOR: SwitchActuatorVirtual,
+}
```

Comment:
Do we need another variable for this? I'd probably keep this within the `FUNCTION_DEVICE_MAPPING` variable. I think adding another variable adds some additional complexities.

Reply:
I don't know how else you want to make the distinction between the "Function Device Mapping" of normal devices and the "Function Device Mapping" of virtual devices (they share the same functionIDs but need different classes). As you see in my implemented examples with this setup we can easily control which virtual devices we support.